tuna/update_golden.py

```diff
@@ -274,6 +311,8 @@ def main():
   total = process_merge_golden(dbt, args.golden_v, query.all())
 
   LOGGER.info("Merged: %s", total)
+  LOGGER.info('Updating conv perf DB table')
+  create_perf_table(args)
```

Comment:
This isn't a good place to run this. This script could be invoked more than once while creating a new golden_v.
eg If for a solver tune we use MI100,MI210,MI250 during tuning: We would run this script once to re-base from the previous golden version, then 3 more times to import from each tuning session.
This table would be created 4x and dropped 3x as a result.

Reply:
OK, then ill add an argument to create this table. This way we can specify that argument when needed (aka for the last run of the update) and we would not be creating/deleting multiple times. Would that work?